src/test/java/com/hocs/test/glue/DispatchStepDefs.java

```diff
@@ -29,13 +29,15 @@
     SuccessfulCaseCreation successfulCaseCreation;
 
     @When("^I complete the dispatch stage$")
-    public void completeTheDispatchStage(){
-        dataInput.selectTeam1();
-        successfulCaseCreation.clickSessionVariableViaLinkTextAndStoreResultingElement();
-       // successfulCaseCreation.clickSessionVariableViaLinkText();
+    public void completeTheDispatchStage() {
+        homepage.findMyDispatchCase();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
+        homepage.selectAllocationUserByVisibleText("Danny Large (danny.large@ten10.com)");
+        homepage.selectMyCases();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkTextAndStoreResultingElement();
         dispatch.clickDispatchAcceptRadioButton();
+        page.clickContinueButton();
```

Comment:
try and replace page here with the actual page the test is on at this moment in time, looks to be the dispatch page but i could be wrong.